x-pack/plugins/security_solution/public/management/pages/policy/view/policy_settings_form/components/notify_user_option.tsx

```diff
@@ -150,6 +152,93 @@ export const NotifyUserOption = React.memo(
       [protection]
     );
 
+    const customNotificationComponent = useMemo(() => {
+      if (!userNotificationSelected) {
+        return null;
+      }
+
+      if (CustomNotificationUpsellingComponent) {
+        return <CustomNotificationUpsellingComponent />;
+      }
+
+      if (!isEditMode) {
```

Comment:
Does this ever render? I tested it out and looks like it is valid pre-render so `-` never shows on the screen.

Reply:
It renders for users who do not have RBAC access to edit custom messages.